### ISSUE TITLE
Hamburger Menu doesn't Open when tapped on mobile

### ISSUE THREAD

**Issue by itsitrio** [SelfhostedPro/medicat-website#1]
When tapping or clicking on the hamburger menu while the site is loaded in a responsive manner for phones screen sizes, the menu does not open, thus preventing access to the Docs page.

![GIF of issue, screencaptured from Firefox Developer Tools](https://i.imgur.com/MBviiDp.gif)

**Comment by SelfhostedPro** [comment]
https://github.com/surjithctly/astro-navbar has usage info to fix that. Will make a fix in the morning. In the meantime /docs is the path to documentation.

**Comment by SelfhostedPro** [comment]
Was actually an issue with that module not working on mobile. Was able to overwrite it. https://github.com/SelfhostedPro/medicat-website/commit/e52d9bfd98ab3af52ebe845a55b85fe0341476b8 should fix it once deployed.

**Comment by itsitrio** [comment]
Sounds good. I've forked the main branch as well. I had started building a docs site ([@itsitrio/MedicatDocs](https://github.com/itsitrio/MedicatDocs) or [docs.medicat.dev](https://docs.medicat.dev)), so I will start working on getting docs imported, and issue a pull once I have all my stuff copied over.

The docs page has some formatting issue when viewed on mobile here, so I can see if I can try and fix that up.
